hostprocess/flannel/flanneld/start.ps1

```diff
@@ -2,7 +2,8 @@ $ErrorActionPreference = "Stop";
 
 # flannel uses host-local, flannel.exe, and sdnoverlay so copy that to the correct location
 Write-Output "Copying SDN CNI binaries to host"
-Copy-Item -Path "$env:CONTAINER_SANDBOX_MOUNT_POINT/cni/*" -Destination "c:\opt\cni\bin" -Force
+mkdir -force $env:CNI_BIN_PATH
```

Comment:
Where does `CNI_BIN_PATH` get defined?

Reply:
flannel-overlay has in it 
`- name: CNI_BIN_PATH`
     `value: C:\\Program Files\\containerd\\cni\\bin`
Same happens for kube-proxy and calico yml files